route-snapper/lib.js

```diff
@@ -384,6 +402,9 @@ export class RouteSnapper {
   #redraw() {
     if (this.loaded) {
       let gj = JSON.parse(this.inner.renderGeojson());
+      if (gj.length == 0) {
```

Comment:
Fairly sure this is a bug. We might have been drawing something previously, then the user makes a change like deleting a waypoint, and there's no longer anything valid to draw. In that case, we still want to update MapLibre and stop drawing things.